sources.cfg

```diff
@@ -4,3 +4,6 @@ extensions = mr.developer
 
 auto-checkout =
   ftw.publisher.core
+
+[branches]
+ftw.publisher.core = dj_abbreviate_constrained_types_when_poss
```

Comment:
Please remove this and wait for `ftw.publisher.core` was merged. I don't think there are circular dependencies.

Reply:
sure - will do, but I needed it for development and proving tests pass for review, right?